### PR TITLE
Stop the change dimension sound

### PR DESCRIPTION
**Pull request by LucGamesYT**

### CODE REVIEW

**Comment by Xernium** [comment]
That is to say I dont have any actual authority to decide that this should be merged. That's up to yesdog. You may need to revise it in the end

**Comment by LucGamesYT** [comment]
Thank you, yes I must have.

**Comment by LucGamesYT** [comment]
It works great for me.

**Comment by TobiasG-DE** [comment]
I like it too, but also no perm to decide that :C

File: BungeeCord-Patches/0065-Stop-the-change-dimension-sound.patch [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ import net.md_5.bungee.util.ChatComponentTransformer;
++import network.ycc.waterdog.pe.packet.PEStopSoundPacket;
+ 
+ @RequiredArgsConstructor
+ public final class UserConnection implements ProxiedPlayer
+@@ -170,6 +171,7 @@ public final class UserConnection implements ProxiedPlayer
+                     net.md_5.bungee.protocol.ProtocolConstants.isPE(getPendingConnection().getVersion())) {
+                 // TODO: find a better place for this
+                 network.ycc.waterdog.pe.PERawPacketData.injectForcedDimChange(ch.getHandle(), getPendingConnection().getVersion());
++                unsafe.sendPacket( new PEStopSoundPacket( "portal.travel", true ) );
```
Comment (colinrgodsey) [inline BungeeCord-Patches/0065-Stop-the-change-dimension-sound.patch]: theres a better place for this actually, but overall, not bad!

**Comment by colinrgodsey** [comment]
will pull this in, clean up, and merge. thanks for the pr!

**Pull request merged**